lib/ui/painting.dart

```diff
@@ -3224,6 +3224,22 @@ abstract class ImageFilter {
     return _GaussianBlurImageFilter(sigmaX: sigmaX, sigmaY: sigmaY, tileMode: tileMode);
   }
 
+  /// Creates an image filter that dilates each input pixel's channel values
+  /// to the max value within the given radii along the x and y axes.
```

Comment:
Is there a better description than "dilates each input pixel's ...". You mentioned a similarity with blur - does it blur them over that radius? Does it add them? Or does it propagate/duplicate them? And how does that action differ for erode? Is erode a subtraction?

Reply:
Well I copied the comments from skia's doc since I think that is the most brief and official.

Reply:
For the record, Flutter has a different audience than Skia. In this case our overall ImageFilter documentation is pretty sparse overall anyway, so this should probably be addressed  in a sweep.

Flutter developers are probably more aware of CSS or SVG specs and concepts than "a famous computer vision library".